src/buildPresets/buildPresets.ts

```diff
@@ -95,6 +95,15 @@ export const buildPresets: IBuildPreset[] = [
         port: 3000,
         group: 'framework'
     },
+    {
+        id: 'html',
+        displayName: 'HTML',
+        appLocation: '/',
+        apiLocation: '',
+        outputLocation: '/',
+        port: 3000, // port not currently used for HTML preset
```

Comment:
What's the port used for? Debugging? The user never gets prompted for it as far as I can tell.

Reply:
Yeah it's for debugging.